scripts/ci/generate_keystores.sh

```diff
@@ -0,0 +1,31 @@
+#! /bin/bash
+#! /bin/bash
+
+# SETTINGS
+# Alfresco Format: "classic" / "current" is supported only from 7.0
+ALFRESCO_FORMAT=current
+
+#Contains directory settings
+source ${GITHUB_WORKSPACE}/alfresco-ssl-generator/ssl-tool/utils.sh
```

Comment:
Please avoid such dependency (`GITHUB_WORKSPACE`) on the execution environment. The better approach would be to set some variable (with a meaningful name) before executing the script.

Reply:
I'm going to unify that script in https://alfresco.atlassian.net/browse/ACS-5005
Now it scattered across 5 repositories.

Reply:
It's not about unifying the key generations but it's rather about the dependency on the runtime environment.

Reply:
Okey, I will also deal with that runtime dependency in others repository.